week-1-practise/level-1/9-function-output-2.js

```diff
@@ -1,14 +1,20 @@
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
-var result = s('code', 'your');
+var result = s("code", "your");
 console.log(result);
 
 // Answer these questions:
 // - What does this program do? Run it and see, or Google some of the methods used ot understand them.
+// This program will join together two or more strings.
 // - How many parameters does the function take?
+// The funtion takes 2 parameters.
 // - What is the function name?
+// the function name is s.
 // - Where is the function called? with what parameters?
+// The function is called at line 5.
```

Comment:
That's right. What about the parameters it is called with?

Reply:
Sorry, i forgot to mention the arguments.